algorithms/CPlusPlus/Linked-Lists/singly.cpp

```diff
@@ -1,4 +1,7 @@
 #include <iostream>
+#include <exception>
+
+using dType = int;
 
 template <typename T>
 class Node {
```

Comment:
don't you think this class should be a member of the list class (`SinglyLinkedList`) ?

so that even if we are accessing it outside the class scope, we access it as `SinglyLinkedList::Node` which will help in self-documenting the program.